myhpi/core/templatetags/core_extras.py

```diff
@@ -2,7 +2,8 @@
 
 register = template.Library()
 
+# logic for this was moved to context.py
 
-@register.filter
-def menu_children(page):
-    return page.get_children().in_menu()
+# @register.filter
+# def menu_children(page):
+#    return page.get_children().in_menu().specific()
```

Comment:
Comments still in code (remove if code is not needed any more)

Reply:
have been removed